tests/integration/test_using_cached_dependencies.py

```diff
@@ -464,3 +472,23 @@ def update_main_repo(env_data, repo_dir, tmpdir, new_dep_commits, dep_repo):
             "FIRST_DEP_COMMIT": new_dep_commits[0],
             "DEP_VERSION": dep_version,
         }
+    elif env_data["pkg_managers"] == ["npm"] or env_data["pkg_managers"] == ["yarn"]:
+        test_dependency = '"assertion-error": "1.1.0"'
```

Comment:
Can we move it out of the test code into `cached_dependencies.yaml` file?

Reply:
Done!